lib/Entity/Video.php

```diff
@@ -429,4 +434,20 @@ public function getCorporateTubeMetadata(): ?CorporateTubeMetaData
     {
         return $this->corporateTubeMetadata;
     }
+
+    public function setAudioOnly(bool $audioOnly): self
+    {
+        $this->audioOnly = $audioOnly;
+
+        return $this;
+    }
+
+    public function isAudioOnly(): bool
+    {
+        if (!$this->audioOnly) {
```

Comment:
that's the same as
`return $this->audioOnly ?? false;`

Reply:
@robertszeker yo, sounds good with the short form.

@GoranDimitrovski we don't save it to DB. This class is used to parse VMPro response and convert to this object.
So, I assume VmPro returns always this key `audioOnly` with false/true.